Log Viewing/ARKLogTableViewController.m

```diff
@@ -71,6 +75,23 @@ - (instancetype)init;
     return [self initWithLogStore:[ARKLogDistributor defaultDistributor].defaultLogStore logFormatter:[ARKDefaultLogFormatter new]];
 }
 
+- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil;
+{
+    return [self init];
+}
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder;
+{
+    NSAssert(NO, @"Please use a supported initializer.");
```

Comment:
Same comment as below. Use NS_UNAVAILABLE in the header and don't implement here